### ISSUE TITLE
Apk path is wrong

### ISSUE THREAD

**Issue by osrl** [bamlab/fastlane-plugin-cordova#34]
I run this:
```
    cordova(
      platform: 'android'
    )
    puts 'apk path:' + ENV['CORDOVA_ANDROID_RELEASE_BUILD_PATH']
```
Here is the output:
```
[18:04:13]: ▸ Built the following apk(s):
[18:04:13]: ▸ /Users/osrl/Documents/web_apps/test/platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk
[18:04:13]: apk path:./platforms/android/build/outputs/apk/android-release.apk
```

**Comment by janpio** [comment]
The APK paths generated by the plugin are for Cordova-Android <7 and haven't been updated yet. Doing this would actually be not that super simple, as then all users of older cordova-android versions would get a broken path :/

**Comment by janpio** [comment]
No, the solution is probably to update this plugin - somehow find out which cordova-android version is in use - and then switch the output path depending on this. Right now it is just generated I think.

**Comment by Almouro** [comment]
Hello guys, a new version has been published as `2.0.0` using this new path
